inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/listener/StartupSortListener.java

```diff
@@ -81,6 +81,12 @@ public ListenerResult listen(WorkflowContext context) throws Exception {
             kvConf.putAll(result);
         }
 
+        String sinkCount = kvConf.get(InlongConstants.SINK_COUNT);
```

Comment:
I think maintaining this config is complex.
Can we use `GroupResourceProcessForm#streamInfos` to determine whether the `sinkList` of `InlongStreamInfo` is empty?